R/xml_find.R

```diff
@@ -203,3 +207,13 @@ xml_find_lgl.xml_nodeset <- function(x, xpath, ns = xml_ns(x)) {
 xml_find_lgl.xml_missing <- function(x, xpath, ns = xml_ns(x)) {
    logical(0)
 }
+
+# Deprecated functions ----------------------------------------------------
+
+#' @rdname xml_find_all
+#' @usage NULL
+#' @export
+xml_find_one <- function(x, xpath, ns = xml_ns(x)) {
+  .Deprecated("xml_find_first")
```

Comment:
Also the `xml_attrs()` change to output namespaces as well will likely break some packages, so we may need to do the release announcement regardless...